elichika/tests/model/MyLSTM.py

```diff
@@ -44,6 +44,17 @@ def forward(self, xs, h, c, mask):
         #h = self.initial_h
         #c = self.initial_c
         inputs = F.pad_sequence(xs)
+        x = None
```

Comment:
Could you add a TODO comment to explain this is a workaround for the bug and these assignments should be removed?